### PR TITLE
Updating Jenkins/Docker/Kube/Plugins and installing JDK

### PR DESCRIPTION
**Pull request by Sheldons92**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,31 +1,23 @@
 FROM quay.io/ukhomeofficedigital/centos-base:v0.5.0
 
-RUN yum install -y wget && wget http://dl.fedoraproject.org/pub/epel/7/x86_64/e/epel-release-7-8.noarch.rpm && \
-  rpm -ivh epel-release-7-8.noarch.rpm
+RUN yum install -y wget &&  wget http://dl.fedoraproject.org/pub/epel/7/x86_64/e/epel-release-7-9.noarch.rpm 
+RUN rpm -ivh epel-release-7-9.noarch.rpm
 
-RUN yum install -y -q python-pip java-headless fontconfig dejavu-sans-fonts git parallel which; yum clean all; pip install awscli
+RUN yum install -y -q python-pip java-1.8.0-openjdk-devel fontconfig dejavu-sans-fonts git parallel which; yum clean all; pip install awscli
```
Comment (jaykeshur) [inline Dockerfile:6]: Why does it need the nonheadless jdk?

Comment (GJKrupa) [reply Dockerfile:6]: As far as I'm aware there is no headless JDK, java-1.8.0-openjdk-headless is a JRE only.

Comment (jaykeshur) [reply Dockerfile:6]: Do you need a JDK at all? JRE should suffice.

Comment (GJKrupa) [reply Dockerfile:6]: Maven doesn't build without a JDK.  We're trying to simplify our current setup which requires building all our Java images inside a Docker container inside Jenkins.

That's Docker in Maven inside Docker inside Jenkins which itself is inside Docker.

Comment (jaykeshur) [reply Dockerfile:6]: I think I understand what you are trying to do. I wonder if you could do the same with JDK? Ie, supply a path to the java bin, which is a docker run command?

Either way, this look fine....

**Review by jaykeshur — APPROVED** [review]

**Pull request merged**